src/containerUtils.js

```diff
@@ -0,0 +1,54 @@
+import invariant from 'invariant';
+import React from 'react';
+import Relay from 'react-relay';
+
+import BatchedRelayContainerBase from './BatchedRelayContainerBase';
+
+export function createBatchedRelayContainer(
+    childElement,
+    params,
+    routeName,
+    queryBatcher
+) {
+  const {queries, fragments} = queryBatcher.flush();
+  const route = {name: routeName, queries, params};
+
+  class BatchedRelayContainer extends BatchedRelayContainerBase {
```

Comment:
Static class property inheritance doesn't work on IE <= 10, per https://babeljs.io/docs/advanced/caveats/#classes-10-and-below-, so e.g. `getFragments` and `getFragmentNames` won't actually be available on `BatchedRelayContainer`.

One option might be to just make `Component` for the root-most container a POJSO, and make `renderFetched` call a separate function that just returns `childElement`, e.g.

```js
return (
  <Relay.RootContainer
    Component={{fragments, ...batchedRelayContainerBase}}
    route={route}
    renderFetched={() => childElement}
  />
);
```